src/nuclear/boltzmann.jl

```diff
@@ -2,15 +2,23 @@
 """
     VelocityBoltzmann(temperature, masses::AbstractVector, dims::Dims{2})
 
-Generate a Boltzmann of velocities for each degree of freedom.
+Generate a Boltzmann distribution of velocities for each degree of freedom.
 
 # Arguments
 
 * `temperature` - Atomic units or Unitful
 * `masses` - Vector of masses for each atom
 * `dims` - (ndofs, natoms). `natoms` must equal `length(masses)`
+* `centre` - Vector of dimension ndofs that provides centre of mass velocity offset
 """
-function VelocityBoltzmann(temperature, masses::AbstractVector, dims::Dims{2})
-    return UnivariateArray([VelocityBoltzmann(temperature, masses[I[2]]) for I in CartesianIndices(dims)])
+function VelocityBoltzmann(temperature, masses::AbstractVector, dims::Dims{2};  centre = nothing)
```

Comment:
I would suggest haveing the default argument as `centre = zeros(dims)`. Then you don't need the if statement. Should centre be a vector or a matrix? I think it should be a matrix with size == dims so you can pick a different direction for each degree of freedom.